packages/realtime_client/lib/src/types.dart

```diff
@@ -101,7 +101,7 @@ class ChannelFilter {
   }
 }
 
-enum ChannelResponse { ok, timedOut, rateLimited, error }
```

Comment:
I wouldn't remove the enum value as this an exposed type, which may cause compile time error now. Better to mark it as deprecated, like the config option.

Reply:
Good call. Fixed it!